src/main/eo/org/eolang/sys/system.eo

```diff
@@ -0,0 +1,41 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2021-2022 Yegor Bugayenko
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
++architect yegor256@gmail.com
++home https://github.com/objectionary/eo-sys
++package org.eolang.sys
++rt jvm org.eolang:eo-sys:jar-with-dependencies:0.0.0
++version 0.0.0
+
+# System object
+[] > system
+
+  # Get time of the day
```

Comment:
@mximp why do we need this object? seems like a very simplistic wrapper around `syscall`. what for?

Reply:
@yegor256 My idea was to have an anchor for system functions like `gettimeofday`. Otherwise what do you suggest for a place for it?

Reply:
@mximp no need to have this. each `QQ.sys.call` will be made individually by the library that needs it. it will be their problem to do what you are trying to do in this `system.eo` object.

Reply:
@yegor256 Got your point! Initially I assumed there would be some login in EO to make the call happen. With current design it is indeed just a simple wrapper. Will remove it.